### PR TITLE
Add flight scenario when falling grom a window

### PR DESCRIPTION
**Pull request by asobolevsky**
Review my story option. @casey-collab

### CODE REVIEW

File: russian/fly/fly-where.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+Вы с гордостью расправляете крылья и за несколько метров от земли падение останвливается
+и Вы всзмываете вверх.
+
+Вы дышите полной грудью! Свобода! Как это прекрасно! 
+
+Вы осматриваетесь по сторонам и решаете ...
+
+
+[ отправиться в пустыню](../desert/desert.md)
```
Comment (casey-collab) [inline russian/fly/fly-where.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: russian/fly/fly-where.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+[ пройти через джунгли](../jungle/jungle.md)
```
Comment (casey-collab) [inline russian/fly/fly-where.md]: This link is also invalid.

File: russian/fly/fly-where.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+[ покорить гору](../mountain/mountain.md)
```
Comment (casey-collab) [inline russian/fly/fly-where.md]: This link is also invalid.

File: russian/window/win-scream/win-scream.md [hunk 4]
```diff
@@ -8,4 +8,6 @@
 
 Вы понимаете что это конец.
 
+[  Постойте! У меня же есть крылья!](../../fly/fly-wings.md)
```
Comment (casey-collab) [inline russian/window/win-scream/win-scream.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: russian/fly/fly-where.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+Как я мог об этом забыть? Я же птица!
```
Comment (asobolevsky) [inline russian/fly/fly-where.md]: Fixed, @casey-collab

**Comment by asobolevsky** [comment]
Review again please, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :wind_chime:

**Pull request merged**